opencog/moses/representation/instance_scorer.cc

```diff
@@ -77,9 +77,9 @@ composite_score atomese_based_scorer::operator()(const instance &inst) const
 
 	combo_tree tr = _rep.get_candidate(inst, _reduce);
 	Handle handle = _to_atomese(tr);
-	if(_as) {
-		handle = _as->add_atom(handle);
-	}
+//	if(_as) {
+//		handle = _as->add_atom(handle);
+//	}
```

Comment:
Is this still necessary?

Reply:
This is necessary if number node "inf" is not supported on local machines otherwise will return segmentation fault. I guess this should remain commented out until we find a way to resolve "inf" being supported on some machines and not on others.

Reply:
OK. I think we can merge that PR for now.